share/wireplumber/scripts/client/access-muvm.lua

```diff
@@ -0,0 +1,143 @@
+MEDIA_ROLE_NONE = 0
```

Comment:
Should we add a `[*.lua]` section to `.editorconfig`?

Reply:
Probably unneeded, it is a slightly patched version of a stock wp script